infrastructure/smart-contracts/test/Referee.mjs

```diff
@@ -725,6 +725,85 @@ export function RefereeTests(deployInfrastructure) {
 			).to.be.revertedWith("11");
 		});
 
+		it("Check rewards calculations are correct over time", async function () {
+			const {
+				referee, 
+				challenger, 
+				mockRollup, 
+				refereeCalculations, 
+				xai, 
+				xaiMinter, 
+				addr1
+			} = await loadFixture(deployInfrastructure);
+
+			const tier1 = 1_250_000_000_000000000000000000n;
```

Comment:
This is more of a style choice, but for me the 18 zeroes are hard to verify accuracy.

I'd prefer ```const tier1 =  ethers.parseEther("1_250_000_000");```